server/app/views/applicant/NavigationFragment.html

```diff
@@ -150,8 +150,14 @@
   </svg>
 </span>
 
-<nav th:fragment="guestSessionAlert" th:if="${isGuest and hasProfile}">
-  <section
+<section
+  th:fragment="guestSessionAlert"
+  th:if="${isGuest and hasProfile}"
+  role="alert"
+  aria-live="polite"
+  aria-label="Guest session informational alert"
```

Comment:
**suggestion:** I think we should have the aria-label be text from the messages file, so it can be translated for different languages. [Here](https://github.com/civiform/civiform/wiki/Internationalization-(i18n)) is our wiki page on i18n and I'm also happy to answer any questions.